snapcraft/plugins/copy.py

```diff
@@ -57,6 +64,6 @@ def build(self):
             os.makedirs(os.path.dirname(dst), exist_ok=True)
 
             if os.path.isdir(src):
-                shutil.copytree(src, dst)
+                shutil.copytree(src, dst, symlinks=self.options.follow_symlinks)
```

Comment:
Quick question: If the symlink is pointing outside of the tree, this will create a .snap with dangling symlinks. Right? Reading the docs it sounds like `ignore_dangling_symlinks` only has an affect if `symlinks` is false.